src/index/coinstatsindex.cpp

```diff
@@ -23,11 +23,11 @@ struct DBVal {
     uint64_t transaction_output_count;
     uint64_t bogo_size;
     CAmount total_amount;
-    CAmount total_subsidy;
```

Comment:
b778ac8f2b12f771e5b8ddcadc35ee267e57cfb0: I think the `total_` prefix is more clear (and consistent with `total_amount` which you wanted to keep). So maybe use the second approach @MarcoFalke suggested:

> unspendables are also "total", but don't mention it. Might be good to remove the "total_" here, or add it to unspendables?

More importantly, can you add comments explaining how these values are used? Their cumulative nature is a bit counter intuitive, though it makes sense for rollbacks.

Reply:
Done, changed the names to all use `total_` and added comments explaining each of the new values in `coinstats.h` a bit more clearly.